solution/ui/regulations/eregs-vite/src/views/PolicyRepository.vue

```diff
@@ -86,8 +106,176 @@ const getDocList = async () => {
     }
 };
 
-getDocList();
+// use "reactive" method to make urlParams reactive when provided/injected
+// selectedParams.paramString is used as the reactive prop
+const selectedParams = reactive({
+    paramString: "",
+    paramsArray: [],
+});
+
+// method to add selected params
+const addSelectedParams = (paramArgs) => {
+    const { id, name, type } = paramArgs;
+
+    // early return if the param is already selected
+    if (selectedParams.paramString.includes(`${type}=${id}`)) return;
+
+    // update paramString that is used as reactive prop for watch
+    if (selectedParams.paramString) {
+        selectedParams.paramString += `&${type}=${id}`;
+    } else {
+        selectedParams.paramString = `?${type}=${id}`;
+    }
+
+    // create new selectedParams key for array of objects for selections
+    selectedParams.paramsArray.push({ id, name, type });
+};
+
+// method to remove selected params
+const removeSelectedParams = (paramArgs) => {
+    const { id, type } = paramArgs;
+
+    // update paramString that is used as reactive prop for watch
+    const paramStringArray = selectedParams.paramString.split("&");
```

Comment:
what if I put &subjects=4&subjects=5.  Right now it breaks.  We use the URL as a source of truth and it is unlikely someone would manually type that but if it came to more parameter's could this be an issue? Not something that needs to be resolved but I'm in a big QA mode this second

Reply:
fixed in [9417c5b](https://github.com/Enterprise-CMCS/cmcs-eregulations/pull/970/commits/9417c5b933be8035e332ad6e384acec449e67dab).  If there are duplicate query params, it just uses the first one.